metpy/tests/test_xarray.py

```diff
@@ -84,6 +84,13 @@ def test_units(test_var):
     assert test_var.metpy.units == units('kelvin')
 
 
+def test_units_percent():
+    """Test that '%' is converted to 'percent' """
+    test_var_percent = xr.open_dataset(get_test_data('irma_gfs_example.nc',
+                                                     as_file_obj=False))['Relative_humidity_isobaric']
```

Comment:
This line is too long...

```suggestion
    test_var_percent = xr.open_dataset(
        get_test_data('irma_gfs_example.nc',
                      as_file_obj=False))['Relative_humidity_isobaric']
```